ok-propertysale-be-business-logic/src/main/kotlin/ru/otus/otuskotlin/propertysale/be/business/logic/operations/QuerySetWorkMode.kt

```diff
@@ -0,0 +1,25 @@
+package ru.otus.otuskotlin.propertysale.be.business.logic.operations
+
+import ru.otus.otuskotlin.propertysale.be.common.context.BePsContext
+import ru.otus.otuskotlin.propertysale.be.common.context.BePsContextStatus
+import ru.otus.otuskotlin.propertysale.be.common.models.common.PsWorkMode
+import ru.otus.otuskotlin.propertysale.mp.pipelines.IOperation
+import ru.otus.otuskotlin.propertysale.mp.pipelines.operation
+
+object QuerySetWorkMode : IOperation<BePsContext> by operation({
```

Comment:
Просто хочу подсказать, что без делегата все то же самое будет:
```suggestion
object QuerySetWorkMode : IOperation<BePsContext> = operation({
```

Reply:
И еще один вариант, который мы в последнее время предпочитаем
```kotlin
fun OperationBuilder<BePsContext>.querySetWorkMode() = operation {
}

val pipeline = pipeline<BePsContext> {
    querySetWorkMode()
}
```